configTools/src/test/scala/com/gu/janus/policy/IamSpec.scala

```diff
@@ -0,0 +1,269 @@
+package com.gu.janus.policy
+
+import com.gu.janus.policy.Iam._
+import io.circe.JsonObject
+import io.circe.parser.decode
+import io.circe.syntax._
+import org.scalatest.freespec.AnyFreeSpec
+import org.scalatest.matchers.should.Matchers
+
+class IamSpec extends AnyFreeSpec with Matchers {
+
+  "Action" - {
+    "should encode action name correctly" in {
+      Action("s3:GetObject").asJson.noSpaces shouldBe """"s3:GetObject""""
+    }
+  }
+
+  "Resource" - {
+    "should encode resource id correctly" in {
+      Resource(
+        "arn:aws:s3:::my-bucket/*"
+      ).asJson.noSpaces shouldBe """"arn:aws:s3:::my-bucket/*""""
+    }
+  }
+
+  "Condition" - {
+    "should encode single condition correctly" in {
```

Comment:
As above, I don't think this codec (or the principal one) are actually used as it stands. We could either drop them (and therefore also these tests), or change the `Statement` encoder to defer to other types for the encoding logic.

Reply:
I'll include what this test was covering in the policy test and drop this.

Reply:
Done in 7d52d1e0f6c82ee3037f26cf29b4ca4a883c8f95 and 9813a072e63b9bba5859c50ffe936b29ba674412